app/src/main/java/org/thoughtcrime/securesms/components/RatingManager.java

```diff
@@ -25,6 +27,11 @@
 
   public static void showRatingDialogIfNecessary(Context context) {
     if (!TextSecurePreferences.isRatingEnabled(context)) return;
```

Comment:
```suggestion
    if (!TextSecurePreferences.isRatingEnabled(context) || BuildConfig.PLAY_STORE_DISABLED) return;
```
ok, don't know what the hack i was thinking but the new check should be integrated with the other one

Reply:
just made the change (though haven't tested it yet)